changes/404.doc.rst

```diff
@@ -0,0 +1 @@
+Update CONTRIBUTING to link to Briefcase-specific docs
```

Comment:
Should be past tense, as they'd appear in release notes - e.g., "Updated CONTRIBUTING..."

Also, this PR adds details about Town Crier to the contribution docs; so that should be mentioned.